src/content/erasmus-icin-pasaport-almak.md

```diff
@@ -47,9 +47,9 @@ Benim aklıma takılan soruların bir çoğunu, sizin de aklınıza takılmış
 
 ### Randevunuz sırasında bilmeniz gerekenler
 
-Mutlaka randevu saatinizden önce randevuyu aldığınız NVİ dairesine gidip Sıramatik cihazından sıranızı alın. Eğer yoğunluk yoksa, randevu saatiniz gelmese bile çağrılabilir ve işlerinizi daha erken halledebilirsiniz. Belgelerinizi eksiksiz tamamladığınızdan emin olun!
+Mutlaka randevu saatinizden önce randevuyu aldığınız NVİ dairesine gidip sıramatik cihazından sıranızı alın. Eğer yoğunluk yoksa, randevu saatiniz gelmese bile çağrılabilir ve işlerinizi daha erken halledebilirsiniz. Belgelerinizi eksiksiz tamamladığınızdan emin olun!
```

Comment:
It's actually a special name, so I guess it's best to keep that first letter uppercase.